### PR TITLE
Docs: add tooltips shared content

### PR DESCRIPTION
**Pull request by imatwawana**
- Adds and updates shared files for tooltips content (2)
- Renames `tooltip-mode.md` to `tooltip-options-1.md`
- Adds shared files to appropriate visualizations
- Adds standalone tooltips content to heatmap and xy chart docs
- Adds **Max height** option to **Configure tooltips** page

### CODE REVIEW

File: docs/sources/shared/visualizations/tooltip-options-1.md [hunk 1]
```diff
@@ -11,3 +11,15 @@ When you hover your cursor over the visualization, Grafana can display tooltips.
 - **Hidden -** Do not display the tooltip when you interact with the visualization.
 
 Use an override to hide individual series from the tooltip.
+
+### Values sort order
+
+When you set the **Tooltip mode** to **All**, the **Values sort order** option is displayed. This option controls the order in which values are listed in a tooltip. Choose from the following:
+
+- **None** - Grafana automatically sorts the values displayed in a tooltip.
+- **Ascending** - Values in the tooltip are listed from smallest to largest.
```
Comment (adela-almasan) [inline docs/sources/shared/visualizations/tooltip-options-1.md:20]: `from smallest to largest` - is there a better/broader word we could use here? Feels (to me) like we're just referring to numbers (we can also sort strings). But no strong opinion 😁

**Comment by adela-almasan** [comment]
Should we also mention `Hover proximity`, for time series, trend and candlestick?

**Comment by imatwawana** [comment]
> Should we also mention `Hover proximity`, for time series, trend and candlestick?

We have! Those visualizations are using the shared file [tooltip-options-2.md](https://github.com/grafana/grafana/pull/86553/files#diff-c3ecde36f32e54d81c9fb76f744ea52797412458edb15bfb99597f260b2b2d71) which includes hover proximity.

File: docs/sources/shared/visualizations/tooltip-options-1.md (hunk 1 above)
Comment (imatwawana) [inline docs/sources/shared/visualizations/tooltip-options-1.md:20]: What about this?
```suggestion
- **Ascending** - Values in the tooltip are listed from lowest to highest.
```

Comment (adela-almasan) [reply docs/sources/shared/visualizations/tooltip-options-1.md:22]: Yes!

**Review by adela-almasan — APPROVED** [review]

File: docs/sources/shared/visualizations/tooltip-options-2.md [hunk 2]
```diff
@@ -1,19 +1,36 @@
 ---
 title: Tooltip options
+comments: |
+  There are two tooltip shared files to cover the most common combinations of options.
```
Comment (imatwawana) [inline docs/sources/shared/visualizations/tooltip-options-2.md:4]:
```suggestion
  There are two tooltip shared files, tooltip-options-1.md and tooltip-options-2.md, to cover the most common combinations of options. 
```

**Pull request merged**